apps/betterangels-backend/common/graphql/types.py

```diff
@@ -1,11 +1,33 @@
 from datetime import datetime
-from typing import Optional
+from typing import Optional, Union
 
 import strawberry
 import strawberry_django
-from common.models import Address, Attachment, Location
+from common.models import Address, Attachment, Location, PhoneNumber
+from phonenumber_field.modelfields import PhoneNumber as DjangoPhoneNumber
+from phonenumbers import parse
 from strawberry import ID, auto
 
+PhoneNumberScalar: Union[DjangoPhoneNumber, str] = strawberry.scalar(
+    DjangoPhoneNumber,
+    serialize=lambda v: str(v.national_number),
+    parse_value=lambda v: parse(v, "US"),
+)
+
+
+@strawberry_django.type(PhoneNumber)
+class PhoneNumberType:
+    id: ID
+    number: Optional[PhoneNumberScalar]  # type: ignore
```

Comment:
What is the type error here, why ignored?

Reply:
this is the same error we're ignoring elsewhere in [ClientProfileBaseType](https://github.com/BetterAngelsLA/monorepo/blob/ccab1e629b0f44831e4d15823e95bb1241878f1a/apps/betterangels-backend/clients/types.py#L162) and [ClientContactBaseType](https://github.com/BetterAngelsLA/monorepo/blob/ccab1e629b0f44831e4d15823e95bb1241878f1a/apps/betterangels-backend/clients/types.py#L186). It's a known strawberry/mypy issue we ran into when we implemented django-phonenumber-field